### PR TITLE
Dzumpano assignment2

### PR DESCRIPTION
**Pull request by dzumpano**

### CODE REVIEW

**Review by Jason-Bengtson — COMMENTED** [review]
python i mean c
Sorry didn't compile i attached a screenshot of the wall error. Also you shouldn't have it labeled as a python when the code is in C.

**Comment by dzumpano** [comment]
I forgot to add how to compile, it should be working now if you read the README

File: problems/euler/pseudo-square-root/python/dzumpano/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+We shall call the largest divisor of an integer n that does not exceed the square root of n the pseudo square root (PSR) of n.
+It can be seen that PSR(3102)=47.
+Let p be the product of the primes below 190.
+Find PSR(p) mod 10^16.'
+
+## How To Run
+If you do not yet have a C complier, i would recomend installing MinGW
+Once you have a C complier, run the command:
+$ gcc psr.c -o psr -lm
+$ ./psr.exe
```
Comment (Jason-Bengtson) [inline problems/euler/pseudo-square-root/python/dzumpano/README.md:14]: I believe this should just be ./psr (at least it was on mine)

**Review by Jason-Bengtson — APPROVED** [review]
dzumpo running now


The code runs now, and I get an output. Not 100% sure the output should be 0. I left one comment on the ReadME  about the running the file should be ./psr instead of ./psr.exe I believe. Only other advice would be perhaps an expected output so I know what I'm looking for. Alas, it runs though! so...
+10

**Pull request merged**